### ISSUE TITLE
Bug Can't Write If Adding.

### ISSUE THREAD

**Issue by thekotekjournal** [laferenorg/binaryfile#2]
# Bug Can't Write If Adding.

## Information
I have issue. If i'am save data for first. i won't to save our data again.
But when i using get function. The data like integer give random number.

## Code
```CPP
#include <iostream>
#include <string>
#include <limits>

#include "include/binaryfile.hpp"

int main(int argc, const char* argv[]) {
	std::binaryfile db = "data.db";
	db.init();
	// db.adding_number<int>(20);
	std::cout << db.get_number<int>(2) << std::endl;
	db.save();
	{
		(void)argc;
		(void)argv;
	}
	return 0;
}
```

## Output
```BASH
6689
```

**Comment by jonathanmarp** [comment]
# Thanks For First Issue

**Comment by jonathanmarp** [comment]
# Fixed

## Information
I have been Fixed: `If i'am save data for first. i won't to save our data again.
But when i using get function. The data like integer give random number.`

## Commit
- Fix Bug [20f725e4361e3b7119645f926628749712c7e336]
